components/usage/pkg/stripe/stripe.go

```diff
@@ -51,7 +54,14 @@ func ReadConfigFromFile(path string) (ClientConfig, error) {
 
 // New authenticates a Stripe client using the provided config
 func New(config ClientConfig) (*Client, error) {
-	return &Client{sc: client.New(config.SecretKey, nil)}, nil
+	sc := &client.API{}
+
+	sc.Init(config.SecretKey, stripe.NewBackends(&http.Client{
```

Comment:
@andrew-farries Actually found a cleaner way to do this using https://github.com/stripe/stripe-go/blob/v70.15.0/stripe.go#L847

Reply:
It is weird `Init()` itself doesn't use this method to set them (even if it just pointed to the default http client)